azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/configuration/AppLocalizationProvider.kt

```diff
@@ -3,14 +3,11 @@
 
 package com.azure.android.communication.ui.configuration
 
-internal class AppLocalizationProvider : LocalizationProvider {
-    private lateinit var language: String
-    private var customTranslation: Map<String, String>? = null
+import java.util.Locale
 
+internal class AppLocalizationProvider : LocalizationProvider {
+    private lateinit var locale: Locale
     override fun apply(localeConfig: LocalizationConfiguration) {
```

Comment:
I can't seem to find any usage of this in the project - where is this being used?